### PR TITLE
V2 automation poc

### PR DESCRIPTION
**Pull request by CameronsOdecee**
- Added first automation test that validates internal v2 framework steps
- Added code review requirement to pull requests (blocking merge to master)
- Added checklist README inside e2e to check off each step as we go!
- Hooked up v2 automation pull request to CI 
 -- Aws agent built on the fly (each build)
 -- https://buildkite.com/tabcorp/tabcorp-cucumber-protractor-framework-v2
 -- Tests running on https://github.com/Tabcorp/v2-automation-react-app
- Updated readme with all our existing steps
- Bumped v2 automation 1.0.32

### CODE REVIEW

File: .buildkite/pipeline.sh [hunk 1]
```diff
@@ -1,10 +1,34 @@
-#!/bin/bash
+#!/usr/bin/env bash
+set -euo pipefail
 
-cat << EOM
+QUEUE="keno-web-build-agent"
```
Comment (lisali-tabcorp) [inline .buildkite/pipeline.sh]: Why this is in common framework? It should be in the project repo

File: src/e2e/step-definitions/then/verify-element-value.ts [hunk 2]
```diff
@@ -139,31 +139,7 @@ Then(/^the "([^"]*)" contains the value "([^"]*)"$/, async (elementName: string,
 /* verify the attribute value of an element within another element */
 Then(/^the "([^"]*)" element within the "(1st|2nd|3rd|[0-9]+th)" "([^"]*)" element contains the "([^"]*)" attribute "([^"]*)"$/, async (subelement: string, parentElementIndex: string, mainElementName: string, attributeType: string, attribute: string) => {
   const index = parseInt(parentElementIndex, 10) - 1;
-  let element = await elementHelper().getElementInElementByCss(mainElementName, subelement, 0, true, index);
+  let element = await elementHelper().getElementInElementByCss(mainElementName, subelement, 0,true,index);
   const elementAttribute = await htmlHelper().getAttribute(element, attributeType);
   expect(elementAttribute).to.include(attribute);
 });
-
-Then(/^the "(1st|2nd|3rd|[0-9]+th)" "([^"]*)" (?:option|element|input) contains the text "([^"]*)"$/, async (elementPosition: string, elementName: string, expectedElementText: string) => {
```
Comment (lisali-tabcorp) [inline src/e2e/step-definitions/then/verify-element-value.ts]: Did you move these steps to another file? We should not delete them

File: .buildkite/pipeline.sh (hunk 1 above)
Comment (CameronsOdecee) [reply .buildkite/pipeline.sh]: this is the agent que that the CI pipeline uses.

Comment (lisali-tabcorp) [reply .buildkite/pipeline.sh]: Change to use a buildkite variable, like `QUEUE=$QUEUE`, and set the variable in buildkite settings, so we do not expose `keno` in the public npm

File: src/e2e/step-definitions/then/verify-element-value.ts (hunk 2 above)
Comment (CameronsOdecee) [reply src/e2e/step-definitions/then/verify-element-value.ts]: fixed in latest merge

File: .buildkite/pipeline.sh (hunk 1 above)
Comment (CameronsOdecee) [reply .buildkite/pipeline.sh]: @codetinkerhack tagging E for his thoughts

Comment (CameronsOdecee) [reply .buildkite/pipeline.sh]: oh wait we already define it in buildkite so yeah not required lol

**Comment by mikhail-trusov** [comment]
lgtm

File: .buildkite/pipeline.sh (hunk 1 above)
Comment (CameronsOdecee) [reply .buildkite/pipeline.sh]: fixed! Thanks :)

File: e2e/config/protractor.conf.ts [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+    },
+    device: 'Local test machine',
+    platform: {
+      name: 'Windows'
+    }
+  },
+  customData: {
+    title: 'Run info',
+    data: [
+      { label: 'Project', value: 'Lott Protractor Tests' },
```
Comment (lisali-tabcorp) [inline e2e/config/protractor.conf.ts]: Change this name to something else please, it is just a label in the test report

**Comment by lisali-tabcorp** [comment]
LGTM

**Pull request merged**